test_launch_ros/test/test_launch_ros/actions/test_ros_timer_action.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Tests for the RosTimerAction Action."""
+import sys
+import time
+import threading
+from functools import partial
+
+import launch
+import launch.event_handlers
+
+from launch_ros.actions import RosTimerAction
+
+import rclpy
+from rclpy.clock import Clock, ClockType
+
+from rosgraph_msgs.msg import Clock as ClockMsg
+from builtin_interfaces.msg import Time
```

Comment:
We should declare test dependencies to these packages in the [package.xml](https://github.com/ros2/launch_ros/blob/master/test_launch_ros/package.xml).

Reply:
I added these as well as a dependency on `rclpy`.